src/pages/dashboard/components/menu/menu.style.tsx

```diff
@@ -3,5 +3,6 @@ import styled from '@emotion/styled'
 export const MenuStyled = styled('nav')`
   padding: 10px;
 
-  background: #cccccc;
+  background: #bd1414;
```

Comment:
please do not use direct colors.

instead in file `index.css`, add following:

`:root {
  --ui-color-background-light: #bd1414;
}`

and then you can use it like this:

`background: var(--ui-color-background-light);`

Reply:
Does this definitely apply to this pull request?